docker/runtime/doris-compose/cluster.py

```diff
@@ -412,7 +414,7 @@ def get_add_init_config(self):
 
             if self.cluster.sql_mode_node_mgr:
                 cfg += [
-                    "cloud_instance_id = " + self.cloud_instance_id(),
+                    "cluster_id = " + CLUSTER_ID,
```

Comment:
resource/init_fe.sh,  resource/init_be.sh,  resource/init_cloud.sh,  command.py
these files use instance id = "default_instance_id",  need them to change ?

Reply:
I will remove them in another pr. pipeline is time consuming.